elastic/apm-server
# RUM V3: add context page fields

## Motivation/summary

Collect more data for RUM analytics.

## Checklist


I have considered changes for:

## How to test these changes

Send RUM V3 data with new fields, and see them indexed. 
Send RUM V3 data without new fields, and see that they are not rejected.

## Related issues

Closes elastic/apm-server#3657
